simple_type_field_names.json5

```
@@ -138,6 +159,12 @@
 
   // Network
   "net/minecraft/unmapped/C_idfydwco": "buf", // PacketByteBuf
+  "net/minecraft/unmapped/C_mnjlaveb$C_mlxyrlwl": {
```

```suggestion
  "net/minecraft/unmapped/C_mnjlaveb$C_mlxyrlwl": { // TagNetworkSerialization$NetworkPayload
```
~~How many places is this a parameter or field?~~
Nice catch, this is actually in quite a few places.